warehouse/ingest-scripts/src/main/resources/bin/util/prettyPrintQuery.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+echo "Arguments:"
+echo "<query>"
+echo "<decorator>"
+echo "Decorator can be EmptyDecorator, HtmlDecorator, or BashDecorator"
+
+cd ../../../../../../../contrib/datawave-quickstart/bin/services/datawave/datawave-ingest-install/bin/ingest
+
+findJars () {
+    ls -1 ../../lib/$1-*.jar | sort | paste -sd ':' -
+}
+
+# Delete line 4 of ./ingest-libs.sh: ". ../ingest/ingest-env.sh"
+sed -i '4d' ./ingest-libs.sh
```

Comment:
I see you are trying to be fancy here.  However normally this would not be an issue.  Please remove the sed line here and below.

Reply:
Okay, it was an issue for me but if it's only on my end then its no problem. I'll delete those and you can let me know if it poses a problem on your end